src/Rho.cc

```diff
@@ -175,6 +175,10 @@ void Rho<OrbitalsType>::rescaleTotalCharge()
             (*MPIdata::sout) << " Rescaling factor: " << t1 << std::endl;
             (*MPIdata::sout) << " Num. electrons: " << nel << std::endl;
         }
+        if (fabs(t1 - 1.) > 0.02)
+        {
```

Comment:
Should we print a statement as to why we are aborting?

Reply:
comment added